packages/bot-web-ui/src/stores/transactions-store.js

```diff
@@ -150,11 +150,11 @@ export default class TransactionsStore {
             }
         );
 
-        // Attempt to load cached transactions on client loginid change.
-        const disposeClientLoginIdListener = reaction(
-            () => client.loginid,
-            () => (this.elements = getStoredItemsByUser(this.TRANSACTION_CACHE, client.loginid, []))
-        );
+        // No need to show previous transactions if the user switches account.
+        // const disposeClientLoginIdListener = reaction(
+        //     () => client.loginid,
+        //     () => this.elements = getStoredItemsByUser(this.TRANSACTION_CACHE, client.loginid, [])
+        // );
 
```

Comment:
Removed the reaction, since it was causing the previous transactions being loaded into the transaction panel

Reply:
Can we remove this piece of code if we don't need it anymore

Reply:
@sandeep-deriv can we delete this entirely?